src/components/Editor/EditorMenu.js

```diff
@@ -75,14 +78,17 @@ function getMenuItems(
     event
   );
 
-  const pairedType = isOriginalId(selectedLocation.sourceId)
-    ? L10N.getStr("generated")
-    : L10N.getStr("original");
+  const isOriginal = isOriginalId(selectedLocation.sourceId);
+  const isPrettyTabOpen =
```

Comment:
i like the idea of having an `isPrettyPrinted` variable

Reply:
Yes, we will need both isOriginal and isPrettyPrinted. As they can be different.